app/services/datastore/application_rehydration.rb

```diff
@@ -73,11 +73,11 @@ def income
     end
 
     def dependants
-      parent.means_details&.dependants&.map { |struct| Dependant.new(**struct) } || []
+      parent.means_details&.income_details&.dependants&.map { |struct| Dependant.new(**struct) } || []
```

Comment:
I started looking at this myself before the break, and had extracted this out into `app/services/adapters/structs/income_details.rb` and then removed `dependants: dependants,` from above because this is then a part of the income object in apply as well as the datastore

Reply:
You were right all along, sorry @hiboabd !